### ISSUE TITLE
Create a new release

### ISSUE THREAD

**Issue by boreq** [ssbc/go-secretstream#22]
We most likely want to tag a new release given the fact that we changed the import path?

https://github.com/ssbc/go-secretstream/compare/v1.2.10...master

```
go: github.com/planetary-social/scuttlego/service/domain/transport/boxstream imports
	github.com/ssbc/go-secretstream/boxstream: github.com/ssbc/go-secretstream@v1.2.10: parsing go.mod:
	module declares its path as: go.cryptoscope.co/secretstream
	        but was required as: github.com/ssbc/go-secretstream
```

**Comment by decentral1se** [comment]
@boreq yep, same goes for all the other `go-*` related modules after the URL migration mayhem. Was waiting until `go-ssb` test suite stabilised to release the tags. But if you need a tag now, can do some tag labours! Lemme know.

**Comment by boreq** [comment]
I don't think it is necessary for me personally because if someone knows how to fix it (is familiar with go modules) and knows what happened (repos were moved) there is a simple way to fix it. However people unfamiliar with the ecosystem may be confused.
